org.mwc.debrief.legacy/src/Debrief/Wrappers/TrackWrapper.java

```diff
@@ -2015,15 +2005,15 @@ public FixWrapper getBacktraceTo(final HiResDate searchTime,
   @Override
   public final WorldArea getBounds()
   {
-    if (cachedBound == null)
+    _bounds = null;

```

Comment:
I'm sure we shouldn't be clearing the cache for each `getBounds()` 😆

Reply:
oh yes. I did the commit while I was doing the performance test. :(